### ISSUE TITLE
MultiSelect option set migration attributes created as option set

### ISSUE THREAD

**Issue by jynxeh** [daryllabar/DLaB.Xrm.XrmToolBoxTools#327]
AttributeManager V 1.2020.10.2

When working with a multi select option set the temporary attribute which is created is not of the multi select type, it is a normal option set.

1. Select an attribute which is of type multi select option set using a global option set
2. Supply a new name for the attribute (in this example I added _new), check the Convert Attribute Type tick box. Leave the Option set as the same global option set.
3. Execute the steps
4. The first step will work, but the following step will fail as it can't find a suitable field.

Expected behaviour is the field should have been created with the same type as the original

![image](https://user-images.githubusercontent.com/8595369/178610329-aba6465e-c27f-4d19-9bdc-31ec30fa8187.png)

I did try to work through the code to see where this might be but having not seen it before I didn't get far. Happy to have a look myself if someone can direct me.

Thanks.

**Comment by daryllabar** [comment]
I don't believe I ever added the code to handle Multi-Select Option Sets vs Option Sets.  Simple work around is to manually delete the _new field, and recreate it as a Multi-Select, and then re-run the tool, skipping just the create temp field step.  You'll probably have to redo it when it comes to recreating it for real as well....

**Comment by jynxeh** [comment]
Hi @daryllabar - thanks for the quick response, sorry I should have said I've worked around it for now, so no problem, just felt like something worth tracking to be fixed/extended at some point.

**Comment by daryllabar** [comment]
Yep... It's on the To do list... Just not sure when it will get moved to the To Done.
